pkg/store/mysql.go

```diff
@@ -36,6 +36,7 @@ var (
 				branch VARCHAR(255),
 				sha VARCHAR(255),
 				identifier BIGINT,
+				actor VARCHAR(255),
```

Comment:
This is a migration list, please add an element to the list with an alter table statement, otherwise this column won't be available for existing installations.